Makefile

```diff
@@ -212,6 +213,8 @@ dist/agent-darwin-amd64:
 	@CGO_ENABLED=1 GOOS=darwin GOARCH=amd64; $(seego) build $(CGO_FLAGS) -o $@ ./cmd/agent
 dist/agent-windows-amd64.exe:
 	@CGO_ENABLED=1 GOOS=windows GOARCH=amd64; $(seego) build $(CGO_FLAGS) -o $@ ./cmd/agent
+dist/agent-windows-installer.exe: dist/agent-windows-amd64.exe
+	cp ./dist/agent-windows-amd64.exe ./packaging/windows && docker build ./packaging/windows -t windows_nsis && docker run --rm -t -v  $(CURDIR)/dist:/app windows_nsis
```

Comment:
I don't think it's required to do this all in a single line. Might make it clearer to split it up.

Reply:
+1, I'd like this split up